### DISCUSSION TITLE
folding (of ts files) through WildWebDeveloper

### DISCUSSION THREAD

**Discussion by jcompagner** [eclipse/tm4e#742] [General]
with the latest releases (not sure when it really happend)
its quite annoying to code typescript files and i think it is because of folding

if i just download eclipse sdk 4.31 and then install only WWD (which i think will also install T4ME)

and i open a ts file then parts are folded for example the import statements

but when i am then start coding the folding is constantly reapplied... (i think this is it)

so the whole editor constantly blinks

Even worse in my normal eclipse install (much older, lot more plugins an constantly upgraded) there i can sometimes be on a completely different position, so the caret is even moving...

Any idea where this is in, or how can we disable the folding completely (i don't care to much if it is all unfolded)

**Comment by sebthom** [comment]
tm4e does not apply folding. this is mostly a bug in lsp4e

**Comment by sebthom** [comment]
Btw. I reported a similar issue https://github.com/eclipse/lsp4e/issues/895

**Comment by sebthom** [comment]
This issue has been fixed in LSP4E
